templates/data/incubator-buildpack-data.yaml

```diff
@@ -263,6 +263,29 @@ templates:
               pipeline.installer: reconciler
               pipeline.test: "fast-integration"
               pipeline.clusterprovisioning: reconciler
+          reconciler_image_autobump_jobConfig:
+            image: "eu.gcr.io/kyma-project/test-infra/kyma-integration:v20211201-ebdeacd8"
```

Comment:
Please use the latest kyma-integration image by adding integration-image global `inheritedConfigs`.

Reply:
Waiting for this PR: https://github.com/kyma-project/test-infra/pull/4454 to be merged, so that I have the latest kyma-integration image on global config.

Reply:
done